### DISCUSSION TITLE
on_hover effect stays even after cursor has left the control.

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#1009] [Q&A] [answered]
I have created a Container and I have given it a hover effect which increases the size of the button and changes its color, the problem is that when I hover the size and color change as expected but when I leave the button the hover effect still stays, and it does not return to the default size and color, is there something I'm doing wrong? Should any additional info be needed I shall provide it ASAP.
Any help would be appreciated.

**Comment by gbrandt** [comment, marked as answer]
On_hover event `data` field is 'true' or 'false' depending on hover state.
